scripts/customize_conda.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+# This customization-script can be used to customize the conda environment.
+# It expects the following metadata:
+#   conda-component: (Required) Must be either ANACONDA or MINICONDA3.
```

Comment:
Add a brief comment on the difference between the two, how to choose, etc. Some users are not familiar with the difference.

Reply:
I have added a link to miniconda3 documentation. It briefly says why to choose miniconda over anaconda.